recipes/rabbitsketch/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set version = "0.1.1" %}
+package:
+  name: rabbitsketch
+  version: {{ version }}
+
+source:
+  url: https://github.com/RabbitBio/RabbitSketch/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 3091e18804d84070b9a453b724858b30c1544be403007e54cfb2e526ecb14f20 
+
+build:
+  number: 0
+  skip: True  # [osx]
+  run_exports:
+    - {{ pin_subpackage('rabbitsketch', max_pin="x.x") }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - python >=3.7,<3.11
+    - cmake
+    - llvm-openmp
```

I made a comment elsewhere. On Conda Forge, OpenMP should use `libgomp` on Linux platforms, and this should be in `build`:

```suggestion
    - libgomp  # [linux]
  host:
    - python >=3.7,<3.11
    - cmake
```